tasks.py

```diff
@@ -216,11 +217,14 @@ def dev_env(ctx, architecture="amd64", name="kind", cni=None, protocol=None):
             config["networking"] = {
                 "disableDefaultCNI": True,
             }
+        extra_options = ""
+        if node_img != None:
+            extra_options = "--image={}".format(node_img)
         config = yaml.dump(config).encode("utf-8")
         with tempfile.NamedTemporaryFile() as tmp:
             tmp.write(config)
             tmp.flush()
-            run("kind create cluster --name={} --config={}".format(name, tmp.name), pty=True, echo=True)
+            run("kind create cluster --name={} --config={} {}".format(name, tmp.name, extra_options), pty=True, echo=True)
```

Comment:
Why did you add "extra_options" instead of just "--image={}"?

Reply:
because otherwise it wouldn't work if we don't specify the image.
I am open to suggestions, my python-fu is pretty bad :-)

Reply:
oh that makes sense. Im just as bad but what you have done makes sense to me